### PR TITLE
Update configuration service

### PR DESCRIPTION
**Pull request by stephanebenayoun**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1105,13 +1105,21 @@
     lodash "^4.17.11"
     to-fast-properties "^2.0.0"
 
-"@capsulajs/capsulajs-configuration-service@^0.1.14", "@capsulajs/capsulajs-configuration-service@latest":
+"@capsulajs/capsulajs-configuration-service@^0.1.14":
```
Comment (dmitriyslaym) [inline yarn.lock]: why do we also have this version installed?

Comment (stephanebenayoun) [reply yarn.lock]: I don't see it in use in any package, not clear why yarn kept it here

Comment (dmitriyslaym) [reply yarn.lock]: It might be the inner dependency of some external package

Comment (dmitriyslaym) [reply yarn.lock]: @capsulajs/environment-registry has this dependency

**Pull request merged**